pkgs/by-name/gr/grmon/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, nix-update-script
+}:
+
+buildGoModule rec {
+  pname = "grmon";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "bcicen";
+    repo = "grmon";
+    rev = "v${version}";
+    hash = "sha256-0J7f4DMADUut3Da0F1eTDsT1Hlk0rfInwzbcVcQNzg8=";
+  };
+
+  vendorHash = "sha256-ySgWEGHlEJpfB/BZuRs1bELBspEaiaX/UnJai2V/hx0=";
+
+  CGO_ENABLED = "0";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-buildid="
```

Comment:
Shouldn't setting buildid be part of buildGoModule?

Reply:
Yes, IMO it should. I'll open a PR to add this to buildGoModule. :)

Reply:
Removed all the ldflags and opened https://github.com/NixOS/nixpkgs/pull/276685